jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -228,18 +238,21 @@ def _get_path(self, path: str) -> str:
             return ""
         return str(Path(*Path(path).parts[1:]))
 
-    # returns None for empty path, i.e. the top-level directory
-    def _get_file_from_path(self, path: str, branch: pfs.Branch) -> pfs.File:
+    # returns None for empty path, i.e. the top-level directory or if no branch is mounted
+    def _get_file_from_path(self, path: str) -> pfs.File:
+        if not self.mounted_branch:
+            return None
+
         name = self._get_name(path)
         if not name:
             return None
 
         path_str = self._get_path(path)
-        file_uri = f"{branch.as_uri()}:/{path_str}"
+        file_uri = f"{self.mounted_branch.as_uri()}:/{path_str}"
         return pfs.File.from_uri(file_uri)
 
-    def download_file(self, path: str, branch: pfs.Branch):
-        file = self._get_file_from_path(path=path, branch=branch)
+    def download_file(self, path: str):
+        file = self._get_file_from_path(path=path)
```

Comment:
file could be None here, but I don't think _download_file is expecting file=None. We should catch and error

Reply:
Agreed, will change this.